templates/puppet/provider.erb

```diff
@@ -186,8 +186,8 @@ Puppet::Type.type(:<%= object.out_name -%>).provide(:google) do
       ['result = new({ title: name,',
        'ensure: :present }.merge(fetch_to_hash(fetch)))'].join(' ')
     end,
-    ('result.instance_variable_set(:@fetched, fetch)' if \
-       object.save_api_results?(config)),
+    ('result.instance_variable_set(:@fetched, fetch)' \
+       if object.save_api_results?(config)),
```

Comment:
no need to pass the `config` once you remove it like I suggested above

Reply:
I did not remove it to avoid a MM-wide change (that affects A/C providers)

Reply:
Consider adding a TODO as a reminder to remove it once the migration is complete.